frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1112,129 +1117,150 @@ function ContextMenuInner(propsWithInputRef: Props) {
-    // maybeClickedMeshId is only set, when a mesh is hovered in the 3d viewport.
-    // Thus the segment id is always unambiguous / clearly defined.
-    if (segmentIdAtPosition > 0 || maybeClickedMeshId != null) {
-      const segmentId = maybeClickedMeshId ? maybeClickedMeshId : segmentIdAtPosition;
-      infoRows.push(
-        <InfoMenuItem key="copy-cell">
+  // Currently either segmentIdAtPosition or maybeClickedMeshId is set, but not both.
+  // segmentIdAtPosition is only set if a segment is hovered in one of the xy, xz, or yz viewports.
+  // maybeClickedMeshId is only set, when a mesh is hovered in the 3d viewport.
+  // Thus the segment id is always unambiguous / clearly defined.
+  if (segmentIdAtPosition > 0 || maybeClickedMeshId != null) {
+    const segmentId = maybeClickedMeshId ? maybeClickedMeshId : segmentIdAtPosition;
+    infoRows.push(
+      getInfoMenuItem(
+        "copy-cell",
+        <>
           <div className="cell-context-icon" />
           Segment ID: {`${segmentId}`} {copyIconWithTooltip(segmentId, "Copy Segment ID")}
-        </InfoMenuItem>,
-      );
-    }
-    if (segments != null && maybeClickedMeshId != null) {
-      const segmentName = segments.get(maybeClickedMeshId)?.name;
-      if (segmentName != null) {
-        const maxSegmentNameLength = 18;
-        infoRows.push(
-          <InfoMenuItem key="copy-cell">
+        </>,
+      ),
+    );
+  }
+  if (segments != null && maybeClickedMeshId != null) {
+    const segmentName = segments.get(maybeClickedMeshId)?.name;
+    if (segmentName != null) {
+      const maxSegmentNameLength = 18;
+      infoRows.push(
+        getInfoMenuItem(
+          "copy-cell",
+          <>
             <Tooltip title="Segment Name">
               <i className="fas fa-tag" />{" "}
             </Tooltip>
             <Tooltip title={segmentName.length > maxSegmentNameLength ? segmentName : null}>
               {truncateStringToLength(segmentName, maxSegmentNameLength)}
             </Tooltip>
-          </InfoMenuItem>,
-        );
-      }
-    }
-
-    if (infoRows.length > 0) {
-      infoRows.unshift(
-        <Menu.Divider
-          key="divider"
-          className="hide-if-first hide-if-last"
-          style={{
-            margin: "4px 0px",
-          }}
-        />,
+          </>,
+        ),
       );
     }
+  }
+
+  if (infoRows.length > 0) {
+    infoRows.unshift({
+      key: "divider",
+      type: "divider",
+      className: "hide-if-first hide-if-last",
+      style: {
+        margin: "4px 0px",
+      },
+    });
+  }
 
-    // It's important to not use <NodeContextMenuOptions ...>
-    // or <NoNodeContextMenuOptions ... />
-    // for the following two expressions, since this breaks
-    // antd's internal population of the correct class names
-    // for the menu.
-    overlay =
+  // It's important to not use <NodeContextMenuOptions ...>
```

Comment:
Luckily, the comment is not needed anymore now :tada: Since these functions return JS objects instead of JSX elements, writing `<NodeContextMenuOptions ...>` wouldn't even make sense, anymore.